src/components/button/button.spec.js

```diff
@@ -0,0 +1,364 @@
+import React from 'react';
```

Comment:
I don't think that we need to check if the default props are set correctly in every test?